editor/uid_upgrade_tool.cpp

```diff
@@ -98,7 +98,7 @@ UIDUpgradeTool::~UIDUpgradeTool() {
 }
 
 void UIDUpgradeDialog::popup_on_demand() {
-	const String confirmation_message = TTR("As of Godot 4.4, scenes and resources use UIDs to reference scripts and shaders.\n\nNormally, this update is applied to a single scene or resource once you save it in Godot 4.4 for the first time. If you have a lot of scenes and/or resources, doing this manually may be time-consuming. This tool will update all of the project's scenes and resources at once.\n\nClick \"Restart & Upgrade\" to restart the editor and update all of the scenes and resources in this project. Depending on the project size, it may take several minutes.\n\nNote: Please make sure you have your project backed up before running the tool, to avoid the possibility of data loss. Additionally, make sure to commit all .uid files into version control (and do not add them to ignore-lists like .gitignore).");
+	const String confirmation_message = TTR("As of Godot 4.4, scenes and resources use UIDs to reference scripts and shaders. This upgrade will be done on scenes and resources next save.\n\nTo save time, this tool can open and save each of these project files. Depending on the project size, it may take several minutes.");
```

Comment:
```suggestion
	const String confirmation_message = TTR("As of Godot 4.4, scenes and resources use UIDs to reference scripts and shaders. This upgrade will be done on scenes and resources next time each is saved.\n\nTo save time, this tool can open and save each of these project files. Depending on the project size, it may take several minutes.");
```
Possibly something like this. Current wording seems slightly ambiguous. I'm not sure if it means that the whole project will be upgraded or just the specific files.

Reply:
I asked an AI as I'm not a native English speaker and it gave me this short sentence (in bold).

> As of Godot 4.4, scenes and resources use UIDs to reference scripts and shaders. **Upgrades apply on the next individual save of each scene/resource.**

How is it?

Reply:
> Upgrades apply on the next save of each individual scene/resource.
Seems fine, I'd prefer this so "individual" emphasizes the resource rather than the act of saving